docs/start/_toc.json

```diff
@@ -35,6 +35,10 @@
           "url": "/start/configure-qiskit-local"
         }
       ]
+    },
+    {
+      "title": "Qiskit versioning",
+      "url": "/start/qiskit-versioning"
     }
```

Comment:
I think this makes sense as top-level, rather than "Advanced setup". But I'm curious what others think.

Reply:
I'm not super keen on having it be a top level  page in the start section. We're really trying to minimise the amount of content there. what about moving this page back to the api reference? I remember we briefly discussed that last year, what do you think?

Reply:
Where would that go in `/api`? A new section like `/api/open-source`?